tests/plotting/test_plotting.py

```diff
@@ -2494,6 +2494,17 @@ def test_chart_matplotlib_plot(verify_image_cache):
     pl.show()
 
 
+def test_get_charts_property():
```

Comment:
Remove the `_property` and I would add an extra assert to verify that `chart is pl.renderer.get_charts()[0]`

Reply:
Done, thanks